doc/classes/ProjectSettings.xml

```diff
@@ -2658,6 +2658,21 @@
 		</member>
 		<member name="rendering/rendering_device/staging_buffer/texture_upload_region_size_px" type="int" setter="" getter="" default="64">
 		</member>
+		<member name="rendering/rendering_device/vsync/frame_queue_size" type="int" setter="" getter="" default="2">
+			The number of frames to track on the CPU side before stalling to wait for the GPU.
+			1 may give you the lowest lag/latency but at the high cost of no parallelism between CPU &amp; GPU.
```

Comment:
Could also start with "A value of", but honestly the subject is fairly clear here.
The ampersand is used extremely rarely in the class reference, too, but I wouldn't split hairs over it, if you feel like it looks better.
```suggestion
			[code]1[/code] may give you the lowest lag/latency but at the high cost of no parallelism between CPU and GPU.
```

Reply:
Same reason as the other value, these are clamped to 2 as well at minimum to avoid introducing "footguns" the users could misconfigure.